src/converters/fromZigbee.js

```diff
@@ -42,22 +42,26 @@ const converters = {
             const result = {};
             const dontMapPIHeatingDemand = model.meta && model.meta.thermostat && model.meta.thermostat.dontMapPIHeatingDemand;
             if (msg.data.hasOwnProperty('localTemp')) {
-                result[postfixWithEndpointName('local_temperature', msg, model, meta)] = precisionRound(msg.data['localTemp'], 2) / 100;
+                let value = precisionRound(msg.data['localTemp'], 2) / 100;
+                value = value < -273.15 ? null : value;
```

Comment:
In case value is null, we should not add it to the result. This (and also the `null`s below should change to)

```js
if (value >= -273.15) {
  result[postfixWithEndpointName('local_temperature', msg, model, meta)] = value;
}
``